src/api.cc

```diff
@@ -223,6 +223,10 @@ class LiteInstall : public InstallContext {
       }
     } else if (rc == data::ResultCode::Numeric::kOk) {
       status = InstallResult::Status::Ok;
+      // TODO: Remove once we add device doc updating on the receiving of the "completed" event to DG.
+      // We need to do this hack, i.e. make at least one request for metadata to DG, in order to make DG
+      // update the device document so it's inline with what is installed on the device.
+      client_->updateImageMeta();
```

Comment:
>Is this worth it?
I think yes, unless we do #2 I posted in the slack thread.
If we wanna do the "big" change that @vkhoroz suggested then, most likely, it won't be done before the next release.

If we are fine with "small" change (just update device record on the receiving of the "completed" event) then it can be done before the next LmP release.

Reply:
i say we just do the "small" approach. Also gives us a sort of "fallback" approach for making sure the field is updated

Reply:
What is the timeline for the next LmP release?

I think I can make the "small" approach next week if we're in rush.
And I can commit to making the "full" approach in September.

Reply:
I removed the hack commit from this PR, so we can merge the other useful changes and bump new aklite version in meta-lmp.